torch/_functorch/autograd_function.py

```diff
@@ -171,8 +188,8 @@ def mark_dirty_error(*args, **kwargs):
 #         return x.exp()
 #
 #     @staticmethod
-#     def setup_context(ctx, outputs, x):
-#         y = outputs
+#     def setup_context(ctx, inputs, output):
+#         y = output
```

Comment:
Why the rename from outputs -> output? is it a single output now? Or they are unpacked?

Reply:
It has always been a single output, just updating the name to reflect that. Since we are returning what the user returned from forward as-is, that can sometime be a tuple, depending on what the user returns.

In an earlier version of this PR I made it always pass in a tuple for consistency, but after discussion here https://github.com/pytorch/pytorch/pull/91222#discussion_r1053897064, I decided to revert that change.